session/txn.go

```diff
@@ -431,7 +437,7 @@ func (s *session) getTxnFuture(ctx context.Context) *txnFuture {
 }
 
 // StmtCommit implements the sessionctx.Context interface.
-func (s *session) StmtCommit() error {
+func (s *session) StmtCommit(memTracker *memory.Tracker) error {
 	defer s.txn.cleanup()
```

Comment:
if we follow code logic, this defer maybe need `memTracker.Consume(-int64(txn.Size()))`, before this line we have put all data from stmt buffer into transaction buffer, and this line will cleanup stmt buffer